peakina/helpers.py

```diff
@@ -39,15 +39,19 @@ class TypeInfos(NamedTuple):
 
 
 SUPPORTED_FILE_TYPES = {
-    "csv": TypeInfos(["text/csv", "text/tab-separated-values"], pd.read_csv),
+    "csv": TypeInfos(
+        ["text/csv", "text/tab-separated-values"],
+        read_csv,
+        ["preview", "chunksize"],
```

Comment:
```suggestion
        ["preview"],
```
but careful! You should also ensure our `read_csv` still `wraps()` `pd.read_csv` to validate all kwargs

Reply:
👍🏾 Updated !